bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -459,6 +462,39 @@ def predict(self):
                     self._candidates[class_name].append(c)
 
 
+class YoloHandlerPytorchYolo(YoloHandler):
+    """
+    Using Pytorch to get YOLO predictions
+    """
+    def __init__(self, config, model_path):
+        """
+        Initialization of PytorchYolo
+
+        :param config:
+        :param model_path:
+        """
+        weightpath = os.path.join(model_path, "yolo_weights.weights")
+        configpath = os.path.join(model_path, "config.cfg")
+
+        self.model = models.load_model(configpath, weightpath)
+
+        self._image = None
+
+        super(YoloHandlerPytorchYolo, self).__init__(config, model_path)
+
+    def predict(self):
+        if self._candidates is None or not self._caching:
+            self._candidates = defaultdict(list)
+            boxes = detect.detect_image(self.model, cv2.cvtColor(self._image, cv2.COLOR_BGR2RGB),
+                                        conf_thres=self._confidence_threshold,
+                                        nms_thres=self._nms_threshold)
+            for box in boxes:
+                # x1, y1, x2, y2, confidence, class
+                c = Candidate.from_x1y1x2y2(*box[0:4].astype(int), box[4].astype(float))
+                classname = self._class_names[int(box[5])]
```

Comment:
Not sure whether this step is necessary as the variable names are sufficient in my opinion. But that's just a minor remark.

Reply:
What do you mean? The comment?

Reply:
see suggestion. but really not that important